torch/CMakeLists.txt

```diff
@@ -163,8 +163,9 @@ endif()
 if(USE_XPU)
     include(${TORCH_ROOT}/cmake/public/xpu.cmake)
     append_filelist("libtorch_python_xpu_sources" TORCH_PYTHON_SRCS)
-
+    
     list(APPEND TORCH_PYTHON_COMPILE_DEFINITIONS USE_XPU)
+    list(APPEND TORCH_PYTHON_LINK_LIBRARIES torch::xpurt)
```

Comment:
I guess, cudart is included in a macro, like TORCH_PYTHON_LINK_LIBRARIES, already?
Can we follow them, instead of specifying xpurt here?